vesting/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "vesting",
+  "version": "0.0.1",
+  "repository": "https://github.com/KlimaDAO/klima-subgraph",
+  "license": "MIT",
+  "scripts": {
+    "codegen": "graph codegen",
+    "build": "graph build",
+    "test": "graph test",
+    "create-local": "graph create --node http://127.0.0.1:8020 vesting",
+    "remove-local": "graph remove --node http://127.0.0.1:8020 vesting",
+    "watch-local": "graph deploy --watch --debug --node http://127.0.0.1:8020/ --ipfs http://127.0.0.1:5001 vesting",
+    "deploy-local": "graph deploy --node http://127.0.0.1:8020 --ipfs http://127.0.0.1:5001 vesting",
+    "deploy": "graph deploy --product hosted-service klimadao/vesting"
```

Comment:
Let's make sure we prefix this with `klimadao-vesting` and ideally copy the dynamic deployment commands using `--` options from the `protocol-metrics` subgraph so we can easily deploy to a different location for testing